test/OrchardCore.Tests/Email/EmailTests.cs

```diff
@@ -73,20 +73,6 @@ public async Task SendEmail_WithDisplayName()
             await SendEmailAsync(message, "Your Name <youraddress@host.com>");
         }
 
-        [Fact]
```

Comment:
This equivalent to

https://github.com/OrchardCMS/OrchardCore/blob/ab07490a2d5ae54180d0af3db27c89287991e567/test/OrchardCore.Tests/Email/EmailTests.cs#L9-L25

Reply:
This has a more appropriate name though, since it's not about the To header, so please rename it.

Reply:
I will use the other name